src/apps/answers/api.py

```diff
@@ -853,9 +853,10 @@ async def applet_validate_multiinformant_assessment(
 
         is_valid = True
     except ValidationError as ex:
-        message = ex.error
-        is_valid = False
-        code = ex.code
+        # Raise a 400 Bad Request when a ValidationError occurs
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST, detail={"valid": False, "message": ex.error, "code": ex.code}
+        )
```

Comment:
we can't do this change, it will change the way API return error and can break frontend

Reply:
+1

Reply:
Adressed on last commit